Week2/arrays.js

```diff
@@ -0,0 +1,12 @@
+console.log(`------ 6 ------`);
+
+let myBooks = [];
+console.log(`the value of my array 'myBooks' will be: 'empty'`);
```

Comment:
I think you accidentally added the homework of week 2! But I will ignore it then :)

Reply:
Yes and I have no idea why it happened so. I created a new branch to upload my week 3 homework but when I created the pull request I realized that my previous homeworks were also added. I deleted the pull requsted and do it all over but this time all previous hw added and my week3 hw added twice! Is it maybe because I used the same repository I forked before for the homework week 2?

Reply:
Aha, yeah I also heard from another student that Yash asked to also have the week 2 homework in this PR, so it's fine:) my bad

Reply:
Ok then, thanks :)